test_scripts/Defects/5_0/1178_Core_sends_an_onAppInterfaceUnregistered_notification_when_UnregisterAppInterface_is_sent_from_Mobile.lua

```diff
@@ -0,0 +1,62 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/SmartDeviceLink/sdl_core/issues/1178
+--
+-- Description:
+-- Core sends an onAppInterfaceUnregistered notification when UnregisterAppInterface is sent from Mobile
+--
+-- It seems as though we have an issue with core sending both an onAppInterfaceUnregistered 
+-- notification and an UnregisterAppInterface response when the mobile proxy sends over 
+-- UnregisterAppInterface. This should not be the case, as the notification should only be sent 
+-- if there is an unregistration from the core side that was not caused by the mobile side. 
+-- The notification also has a reason of -1, which is not a valid value.
+-- We should remove the notification from being sent when the UnregisterAppInterface RPC is sent.
+--
+-- Preconditions:
+-- 1) Clear environment
+-- 2) SDL, HMI, Mobile session started
+-- 3) Registered app
+-- 4) Activated app
+--
+-- Steps: 
+-- 1) Send "UnregisterAppInterface" mobile RPC
+--   
+-- Postconditions:
+-- Stop SDL
+--
+-- Expected:
+-- 1) "onAppInterfaceUnregistered" notification was NOT recieved by mobile
+-- 2) "UnregisterAppInterface" response was recieved by mobile with resultCode = "SUCCESS"
+-- 3) "OnAppUnregistered" was recieved by HMI with unexpectedDisconnect = false 
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('user_modules/sequences/actions')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Functions ]]
+local function unregisterAppInterface()
+	local cid = common.getMobileSession():SendRPC("UnregisterAppInterface", { })
+	common.getMobileSession():ExpectNotification("onAppInterfaceUnregistered", { })
```

Comment:
@KrupenichOleg Typo `onAppInterfaceUnregistered` - please take into account that names of reqiuests/notifications re case sensitive